### DISCUSSION TITLE
change github username

### DISCUSSION THREAD

**Discussion by codecunguk** [community/community#69307] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

how to change github username ?

**Comment by Akbaroke** [comment]
To change your GitHub username, follow these steps:

Go to your GitHub profile page.
Click the Settings button.
In the left sidebar, click Account.
In the "Change username" section, click Change username.
Enter your new username and click Change username.
You will be prompted to enter your GitHub password to confirm the change. Once you have confirmed the change, your username will be updated.

Note: Changing your GitHub username will update your username on all of your GitHub repositories and contributions. It will also update your username on any other websites or services that you have linked to your GitHub account.

Here are some things to keep in mind when changing your GitHub username:

Your new username must be unique and cannot already be in use by another GitHub user.
Your new username must be between 3 and 39 characters long.
Your new username can only contain alphanumeric characters, hyphens, and underscores.
You cannot change your username more than once every 180 days.
I hope this helps!

**Reply by codecunguk** [reply]
thank you for helping in this matter..

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
